src/util/ssa_expr.cpp

```diff
@@ -61,18 +61,23 @@ static void build_ssa_identifier_rec(
 
     if(!l0.empty())
     {
+      // Distinguish different threads of execution
       os << '!' << l0;
       l1_object_os << '!' << l0;
     }
 
     if(!l1.empty())
     {
+      // Distinguish different calls to the same function (~stack frame)
       os << '@' << l1;
       l1_object_os << '@' << l1;
     }
 
     if(!l2.empty())
+    {
+      // distinguish assignments to the same variable
```

Comment:
<pedantry>s/distinguish/Distinguish/ for consistency</pedantry>

Also I might phrase this slightly differently as this is not just assignments (instructions in the program) but SSA steps.  One assignment can give many SSA steps, for example if it is in a loop.

"Distinguish SSA steps for the same variable"  maybe?